src/main/java/com/crowdin/cli/commands/functionality/RequestBuilder.java

```diff
@@ -160,9 +160,10 @@ public static ExportProjectTranslationRequest exportProjectTranslation(ExportPro
     }
 
     public static CrowdinTranslationCraeteProjectPseudoBuildForm crowdinTranslationCreateProjectPseudoBuildForm(
-        Boolean pseudo, Integer lengthCorrection, String prefix, String suffix, CharTransformation charTransformation
+        Boolean pseudo, Integer lengthCorrection, String prefix, String suffix, CharTransformation charTransformation, long branchId
```

Comment:
let's the order of the arguments be the same as in the `CrowdinTranslationCraeteProjectPseudoBuildForm` (branch ID first)

also, the `branchId` probably is nullable